docs/Troubleshooting.md

```diff
@@ -17,13 +23,23 @@ or on Windows
 node --inspect-brk ./node_modules/jest/bin/jest.js --runInBand [any other arguments here]
 ```
 
-This will run Jest in a Node process that an external debugger can connect to. Note that the process will pause until the debugger has connected to it.
+This will run Jest in a Node process that an external debugger can connect to. 
+
+:::note
+
+Note that the process will pause until the debugger has connected to it.
+
+:::
```

Comment:
Fine to keep inline.

```suggestion
This will run Jest in a Node process that an external debugger can connect to. The process will pause until the debugger has connected to it.
```